docs/_static/vcr/v1.yaml

```diff
@@ -85,6 +89,12 @@ paths:
             text/plain:
               schema:
                 type: string
+        "409":
```

Comment:
We don't use this error code anywhere. Perhaps must use a 400 to keep the set small?

Reply:
409 is a perfectly valid and widely used code. 
400 suggests that your input was wrong and when fixed, you may try again. That's not the case here. The state of the VC has changed and you shouldn't try again.